app/models/school_cohort.rb

```diff
@@ -5,6 +5,7 @@ class SchoolCohort < ApplicationRecord
     full_induction_programme: "full_induction_programme",
     core_induction_programme: "core_induction_programme",
     design_our_own: "design_our_own",
+    no_early_career_teachers: "no_early_career_teachers",
     not_yet_known: "not_yet_known",
```

Comment:
Is this actually used anywhere? I think this might be a leftover

Reply:
It's used in the dashboard to show what whats chosen for that cohort as well as to record that the school has said there are none.